.travis.yml

```diff
@@ -1,31 +1,48 @@
-# R for travis: see documentation at https://docs.travis-ci.com/user/languages/r
-
-language: R
+language: r
 dist: bionic
-sudo: false
-cache: packages
-latex: true
-r_github_packages: mlr-org/mlr3book
-pandoc_version: 2.7.3
+cache:
+  - packages
+  - ccache
+
+latex: false
+
+addons:
+  apt:
+    packages:
+       - lmodern
+       - texlive-xetex
+       - texlive-math-extra
+       - texlive-base
+       - texlive-binaries
+       - texlive-fonts-extra
+       - texlive-fonts-recommended
+       - texlive-generic-recommended
+       - texlive-humanities
+       - texlive-latex-base
+       - texlive-latex-extra
+       - texlive-latex-recommended
+       - texlive-pictures
+       - texlive-publishers
+       - texlive-science
+
+before_install:
+  - echo -e "options(Ncpus = 8, repos = structure(c(CRAN = 'https://cloud.r-project.org/')))" > $HOME/.Rprofile
+  - mkdir $HOME/.R && echo -e 'CXX_STD = CXX14\n\nCC=ccache gcc -std=gnu99\nCXX=ccache g++\nC11=ccache g++\nC14=ccache g++\nFC=ccache gfortran\nF77=ccache gfortran' > $HOME/.R/Makevars
+  - echo -e 'max_size = 5.0G\nsloppiness = include_file_ctime\nhash_dir=false' > $HOME/.ccache/ccache.conf
+  - R -q -e 'if (!requireNamespace("remotes")) install.packages("remotes")'
+  - R -q -e 'if (getRversion() < "3.2" && !requireNamespace("curl")) install.packages("curl")'
+  - R -q -e 'remotes::install_github("ropenscilabs/tic"); print(tic::dsl_load()); tic::prepare_all_stages()'
+  - R -q -e 'remotes::install_local()' # install mlr3book
+  - R -q -e 'tic::before_install()'
+
+install:
+  - R -q -e 'tic::install()'
```

Comment:
Why do we need tic now? The setup just seems more complicated than what we had before.

Reply:
NB: This comes with additional >50 lines of custom code in tic.R :(

Reply:
Because this is simply the way how I write CI files. 🙂 
The `tic.R` here is extra descriptive to show what is going on, it could be condensed by at least 20 lines.

We do not need it necessarily, so feel free to push all of this back to `.travis.yml`.